osbs/tekton.py

```diff
@@ -26,16 +26,15 @@
 
 logger = logging.getLogger(__name__)
 
-# Retry each connection attempt after 5 seconds, for a maximum of 10 times
+# Retry each connection attempt after 5 seconds, for a maximum of 20 times
 WATCH_RETRY_SECS = 5
-WATCH_RETRY = 10
-# Retry waiting for pipeline to finish for 5 seconds, for a maximum of 10 times
+WATCH_RETRY = 20
+MAX_BAD_RESPONSES = 20
+
+# Retry waiting for pipeline to finish for 5 seconds, for a maximum of 5 hours
 WAIT_RETRY_SECS = 5
-WAIT_RETRY = 10
-MAX_BAD_RESPONSES = WATCH_RETRY // 3
-# Give up after 12 hours
-WAIT_RETRY_HOURS = 12
-WAIT_RETRY = WAIT_RETRY_HOURS * 3600 // (WATCH_RETRY_SECS * WATCH_RETRY)
+WAIT_RETRY_HOURS = 5
+WAIT_RETRY = (WAIT_RETRY_HOURS * 3600) // WATCH_RETRY_SECS
```

Comment:
```suggestion
WAIT_RETRY = (WAIT_RETRY_HOURS * 3600) // WAIT_RETRY_SECS
```

Reply:
I see. The only weird detail is that the formula uses **WATCH**_RETRY_SECS - I'm guessing that should be **WAIT**_RETRY_SECS?

Reply:
right, fixed